Month1/Week2/Section2/Month1-Week2-Section2/Spotify/PlayListContainer.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include "PlayList.h"
+
+
+class FPlayListContainer {
+private:
+	const static int MaxPlayListCount = 5;
+	FPlayList PlayLists[MaxPlayListCount];
+	int PlayListCount;
+public:
+	FPlayListContainer();
+	FPlayListContainer(FPlayList NewPlayList);
```

Comment:
Do you really need a constructor that takes in a playlist for a container of playlists?

Reply:
You havve reason, not necesary I delete that constructor